src/components/element-editor/validation/common/validation-error-message.tsx

```diff
@@ -0,0 +1,39 @@
+import { Form } from "@lifesg/react-design-system/form";
+import { Controller, useFormContext } from "react-hook-form";
+import { TOverallTextBasedSchema } from "src/schemas";
+
+interface IProps {
+    index: number;
+}
+
+export const ValidationErrorMessage = ({ index }: IProps) => {
+    // ===========================================================================
+    // CONST, STATE, REF
+    // ===========================================================================
+    const {
+        formState: { errors },
+        control,
+    } = useFormContext<TOverallTextBasedSchema>();
```

Comment:
instead of `TOverallTextBasedSchema`, we can have a new type to cover all schemas with validation. this way if new elements are added, devs wont miss adding the support to these components